lib/app/dispatcher.rb

```diff
@@ -132,6 +129,25 @@ def start_filter_validation_task(connector_settings)
           Utility::ExceptionTracking.log_exception(e, "Filter validation task for #{connector_settings.formatted} failed due to unexpected error.")
         end
       end
+
+      def start_consumers!
+        @consumer = Core::Jobs::Consumer.new(
+          poll_interval: POLL_INTERVAL,
+          termination_timeout: TERMINATION_TIMEOUT,
+          min_threads: MIN_THREADS,
+          max_threads: MAX_THREADS,
+          max_queue: MAX_QUEUE
+        )
+
+        @consumer.subscribe!(index_name: Utility::Constants::JOB_INDEX)
```

Comment:
The method is using plural, but there's one consumer.

This is an extension point, right? At some point you'll have multiple consumers?

Reply:
Yes, we might be running multiple consumers. I will rename it though